go-controller/pkg/ovn/gress_policy.go

```diff
@@ -37,6 +37,12 @@ type gressPolicy struct {
 	// except the IP block in the except, which should be dropped.
 	ipBlockCidr   []string
 	ipBlockExcept []string
+	ipBlock       *ipBlock
+}
+
+type ipBlock struct {
+	cidr   string
+	except []string
```

Comment:
Really you don't even need this type; you can just use `knet.IPBlock` directly instead

Reply:
done